src/util/svgTagTemplate.mjs

```diff
@@ -0,0 +1,80 @@
+export function svg(string) {
+    const markup = parseFromSVGString(string[0]);
+    return markup;
+}
+
+function parseFromSVGString(str) {
+
+    const parser = new DOMParser();
+    const type = 'text/html';
+    const document = parser.parseFromString(`<svg>${str.trim()}</svg>`, type);
+    const svg = document.querySelector('svg');
+    return build(svg);
+}
+
+function build(root) {
+    const markup = [];
+
+    Array.from(root.children).forEach(node => {
+        const markupNode = {};
+        const { tagName, attributes, textContent, namespaceURI, style } = node;
+
+        markupNode.tagName = tagName;
+        markupNode.namespaceURI = namespaceURI;
+
+        const stylesObject = {};
+        for (var i = style.length; i--;) {
+            var nameString = style[i];
+            stylesObject[nameString] = style.getPropertyValue(nameString);
+        }
+        markupNode.style = stylesObject;
+
+        // selector fallbacks to tagName
+        const selectorAttribute = attributes.getNamedItem('@selector');
+        if (selectorAttribute) {
+            markupNode.selector = selectorAttribute.value;
+            attributes.removeNamedItem('@selector');
+        } else {
+            markupNode.selector = tagName;
+        }
+
+        const groupSelectorAttribute = attributes.getNamedItem('@group-selector');
+        if (groupSelectorAttribute) {
+            const groupSelectors = groupSelectorAttribute.value.split(',');
+            if (groupSelectors.length === 1) {
```

Comment:
I don't think we need to branch it here. This code will work fine with one and more group selectors.
```
markupNode.groupSelector = groupSelectors.map(s => s.trim())
```